.circleci/config.yml

```diff
@@ -149,10 +174,8 @@ workflows:
               only:
                 - /v.*/
       - api-governance:
+          context: api-compliance
           filters:
-            branches:
-              # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
-              ignore: /pull\/[0-9]+/
```

Comment:
Restore this condition
it is not secure to expose the secrets needed to PRs from forks and without them it will always fail

Reply:
done